spec/requests/admin_claim_checks_spec.rb

```diff
@@ -47,11 +45,11 @@
       follow_redirect!
     end
 
-    describe "claim_approvals#create" do
+    describe "claim_checks#create" do
       let(:claim) { create(:claim, :submitted) }
 
       it "does not allow a claim to be approved" do
-        post admin_claim_approvals_path(claim_id: claim.id)
+        post admin_claim_checks_path(claim_id: claim.id, result: "approved")
```

Comment:
This is the test that was passing despite missing the `result` parameter, btw.

Reply:
Ah right, ok. What was the expectation before? It looks right to me now – it verifies specifically an "unauthorized" response, which shouldn't result in a false positive.

Reply:
The test is the same. Ignore me.